Automattic/sensei
# Remove multiple product list from course in Course List block

Part of fixing the issue of product list being visible above the Buy Course button when in Course List block.

### Changes proposed in this Pull Request

* We decided not to render the product list above the Buy Course button when it's in Course List, but it should still render them when the take course button block is used elsewhere, like in the Course Page. To determine when to render the product list or not, we needed to know from inside the button if it's being rendered in a Course List block or not. Ideally, it is done using context and checking for queryId, but it gets overridden due to another issue not being fixable from inside Sensei. So we are adding an attribute to the inner blocks when they are being rendered in a Course List block.

### Testing instructions
- Check out this [branch](https://github.com/Automattic/sensei-pro/pull/2091)
- Make sure the buttons work as expected following the instructions there
